src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +16,7 @@ public interface MachineService {
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<T> getAll(Class type);
```

Comment:
Don't use raw types. Always parametrize interfaces and classes if it is possible. The `getAll` method should take a parameterized Class type as its argument. For example, it should be `Class<T> type` instead of `Class type` to enforce type safety.